classes/class-to-specials-frontend.php

```
@@ -82,12 +87,40 @@ public function price_filter( $html = '', $meta_key = false, $value = false, $be
 		return $html;
 	}
 
+	public function price_type_filter( $html = '', $meta_key = false, $value = false, $before = '', $after = '' ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$after'. (undefined)

(UnusedFormalParameter)